test-data/unit/fixtures/notimplemented.pyi

```diff
@@ -10,5 +9,10 @@ class function: pass
 class bool: pass
 class int: pass
 class str: pass
-NotImplemented = cast(Any, None)
 class dict: pass
+
+class _NotImplementedType(Any):
+    __call__: NotImplemented  # type: ignore
+NotImplemented = _NotImplementedType()
```

Comment:
Nit: What about using `NotImplemented: _NotImplementedType` to better align with typeshed?